src/clientsettingsdlgbase.ui

```diff
@@ -242,6 +242,13 @@
                     </property>
                    </widget>
                   </item>
+                  <item>
+                   <widget class="QLabel" name="lblAudioAlerts">
+                    <property name="text">
+                     <string>Audio Alerts</string>
```

Comment:
--> Audio Alerts (might need secondary sound hardware)

Reply:
Hrm, not sure if there's space for so much text. Have you checked?

Reply:
Maybe we could add a tooltip later - and something clear in the documentation?

Reply:
The *What's this* text does have a hint regarding that already, btw.